src/article/components/ManuscriptComponent.js

```diff
@@ -5,18 +5,22 @@ import ManuscriptSection from './ManuscriptSection';
 export default class ManuscriptComponent extends Component {
   render($$) {
     const manuscript = this.props.model;
+
+    // TODO: Need to understand why in some places they 'lookup' the components and assign them to variables, vs just a
+    //       simple import. Assume there is some method to - what seems to me - madness in doing things different ways.
```

Comment:
This could be because this method throws an exception if the component is not registered.